src/ServiceControl.Monitoring/Metrics/Raw/LongValueOccurenceSerializerDefinition.cs

```diff
@@ -0,0 +1,59 @@
+﻿namespace ServiceControl.Monitoring.Processing.RawData
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using global::NServiceBus.MessageInterfaces;
+    using global::NServiceBus.Serialization;
+    using global::NServiceBus.Settings;
+    using NServiceBus.Metrics;
+
+    class LongValueOccurrenceSerializerDefinition : SerializationDefinition
```

Comment:
Using a serializer requires execution of full pipeline. I think that for the purpose of these messages we would be better off using a behavior in the transport receive pipeline. This way we would shortcut the pipeline saving us some processing cycles.

The behavior would replace both serializer and the handler.